activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Allow postgres errors to be ignored during the loading of db/structure.sql via the `IGNORE_PG_LOAD_ERRORS` ENV.
```

Comment:
- `postgres` --> `PostgreSQL`
- can you wrap db/structure.sql in backticks
- can you add a comma after `.sql`